letsencrypt/client/tests/account_test.py

```diff
@@ -0,0 +1,188 @@
+"""Tests for letsencrypt.client.account."""
+import logging
+import mock
+import os
+import pkg_resources
+import shutil
+import sys
+import tempfile
+import unittest
+
+import zope.component
+
+from letsencrypt.acme import messages2
+
+from letsencrypt.client import account
```

Comment:
this should be imported from within unit test to stay consistent with the rest of the codebase